build_projects/dotnet-cli-build/PrepareTargets.cs

```diff
@@ -148,27 +148,38 @@ public static BuildTargetResult DownloadHostAndSharedFxArchives(BuildTargetConte
             var sharedFrameworkVersion = DependencyVersions.SharedFrameworkVersion;
             var sharedFrameworkChannel = DependencyVersions.SharedFrameworkChannel;
 
-            var combinedSharedHostAndFrameworkArchiveFile = c.BuildContext.Get<string>("CombinedFrameworkHostCompressedFile");
+            var combinedSharedHostAndFrameworkArchiveDownloadFile =
+                Path.Combine(Dirs.CoreSetupDownload, "combinedSharedHostAndFrameworkArchive");
+            var combinedSharedHostAndFrameworkArchiveDestinationFile = 
```

Comment:
I think it would be easier to reason about the code if this was moved inside the `if (!File.Exists` block.  That way it is really easy to tell that the above file is the one being checked for existence.